src/content/docs/style-guide/images/create-videos.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Create videos
+tags:
+  - Tech writer style guide
+  - Processes and procedures
+redirects:
+---
+
+import styleGuideCamtasiaEditing from 'images/style-guide_screenshot-crop_camtasia-editing.png'
+
+import styleGuideCamtasiaBlur from 'images/style-guide_screenshot-full_camtasia-blur.png'
+
+
+There are many ways to create videos, but here are some tips that will likely apply to a lot of video production projects. 
+
+## Before you start [#before-you-start]
+
+Before you start, talk to the Nerd Bytes team and see if your video content relates to something they've already worked on, or plan on working on.
+
+## Some tips on production value [#production-value]
+
+Some thoughts on production value and how long we take on these projects. 
+
+Be okay with some imperfection. We are not video production professionals and we don't believe anyone expects us to be. A very quickly and roughly made video can still show how to use the UI and still benefit people even without high production value.
+
+Another reason we are okay with some lower production quality: we know that most of our videos will go out-of-date sometime in future, sometimes in very near future. For this reason, we see it as better to err on side of getting helpful videos out quickly as opposed to spending a lot of time on videos that we know we'll have to replace.  
+
+Don't let your quest for perfection be at odds with getting out a highly useful video on quicker side. 
+
+To take some common examples of imperfection that can bother producers but that aren't that big of a deal: 
+
+* A quickly written and produced video can be completely fine, provided you think it will add obvious value.  
+* Some rough spots in your voice-over (uhs, umms, pauses) are okay.
+* Some background audio hiss or noise can be okay, if not too bad. 
+* Some rough/imperfect cuts between clips (for example, a cut where a cursor suddenly jumps somewhere else) can be okay. 
+
+## Scripting tips [#script-tips]
+
+Consider talking first to SMEs who know about that UI. Consider talking to UX designers who may know about best practices for a specific UI or about upcoming changes that might make your video out-of-date in near future. 
```

Comment:
@zuluecho9 I'd recommend creating these as bullets. You might have an introductory sentence like this: "You can take a variety of approaches to scripting your video:"